dss_client/CMakeLists.txt

```diff
@@ -57,19 +56,22 @@ target_compile_definitions(test_dss PUBLIC "DSS_DEBUG")
 
 target_include_directories(test_dss PUBLIC 
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/include>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
 target_include_directories(${DSS_LIB} PUBLIC 
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/include>
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
 
 target_link_libraries(test_dss ${AWSSDK_LINK_LIBRARIES})
 target_link_libraries(${DSS_LIB} ${AWSSDK_LINK_LIBRARIES})
-target_link_libraries(test_dss ${EXT_INCLUDE_FLAGS})
-target_link_libraries(${DSS_LIB} ${EXT_INCLUDE_FLAGS})
+target_link_libraries(test_dss "${PYTHON3_LINK_LIBRARIES_FLAGS} ${PYTHON_LIBRARIES}  -L${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/lib -lrdmacm -libverbs -lcrypto -lcurl -lrdd_cl")
+target_link_libraries(${DSS_LIB} "${PYTHON3_LINK_LIBRARIES_FLAGS} ${PYTHON_LIBRARIES} -L${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/lib -lrdmacm -libverbs -lcrypto -lcurl -lrdd_cl")
 
```

Comment:
why we need -lrdmacm -libverbs ? Adding -lrdd_cl should be enough

Reply:
need to see if i can remove that without errors

Reply:
fixed it